test/specs/common/tests/02.column.js

```diff
@@ -0,0 +1,194 @@
+                        var spy = spyOn(formatUtils, 'printFloat').and.callThrough();
+                        var col = table1_schema2.columns.getByPosition(7);
+                        formattedValue = col.formatvalue(11.1);
+                        expect(spy).toHaveBeenCalledWith(11.1, undefined);
+                        expect(formattedValue).toBe('11.10');
+                    });
+
+                    it('float8 columns correctly.', function() {
+                        var spy = spyOn(formatUtils, 'printFloat').and.callThrough();
+                        var col = table1_schema2.columns.getByPosition(8);
+                        var options = {numDecDigits: 7};
+                        formattedValue = col.formatvalue(234523523.023045230450245, options);
+                        expect(spy).toHaveBeenCalledWith(234523523.023045230450245, options);
+                        expect(formattedValue).toBe('234,523,523.0230452');
+                    });
+
+                    afterEach(function() {
+                        expect(formattedValue).toEqual(jasmine.any(String));
+                        formattedValue = undefined;
+                    });
+                });
+
+                describe('should call printInteger() to format,', function() {
+                    var formattedValue = undefined;
+
+                    it('int2 columns correctly.', function() {
+                        var spy = spyOn(formatUtils, 'printInteger').and.callThrough();
+                        var col = table1_schema2.columns.getByPosition(9);
+                        var int2 = 32768;
+                        formattedValue = col.formatvalue(int2);
+                        expect(spy).toHaveBeenCalledWith(int2, undefined);
+                        expect(formattedValue).toBe('32,768');
+                    });
+
+                    it('int4 columns correctly.', function() {
+                        var spy = spyOn(formatUtils, 'printInteger').and.callThrough();
+                        var col = table1_schema2.columns.getByPosition(10);
+                        var int4 = -2147483648;
+                        formattedValue = col.formatvalue(int4);
+                        expect(spy).toHaveBeenCalledWith(int4, undefined);
+                        expect(formattedValue).toBe('-2,147,483,648');
+                    });
+
+                    it('int8 columns correctly.', function() {
+                        var spy = spyOn(formatUtils, 'printInteger').and.callThrough();
+                        var col = table1_schema2.columns.getByPosition(11);
+                        var int8 = 9007199254740991; // Max safe integer in JS
+                        var options = {numDecDigits: 7};
+                        formattedValue = col.formatvalue(int8);
+                        expect(spy).toHaveBeenCalledWith(int8, undefined);
+                        expect(formattedValue).toBe('9,007,199,254,740,991');
+                    });
+
```

Comment:
I added a couple more test cases to address this in [this commit](https://github.com/informatics-isi-edu/ermrestjs/pull/147/commits/7bf8ead12754ada56fed7c6ce8b339ef5770e300).